deps/Versions.make

```diff
@@ -22,7 +22,7 @@ DSFMT_VER := 2.2.4
 DSFMT_JLL_NAME := dSFMT
 
 # GMP
-GMP_VER := 6.2.1
+GMP_VER := 6.2.1+2
```

Comment:
This shouldn't be touched:
```suggestion
GMP_VER := 6.2.1
```
Instead you have to update the version number in the Project.toml of the stdlib

Reply:
Thanks - this is what I was missing.